modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -78,35 +111,41 @@ $ oc create rolebinding shared-resource-my-share --role=shared-resource-my-share
 apiVersion: build.openshift.io/v1
 kind: BuildConfig
 metadata:
-  name: my-csi-bc
-  namespace: my-csi-app-namespace
+  name: entitled-build
+  namespace: entitlement-test
 spec:
   runPolicy: Serial
   source:
     dockerfile: |
-      FROM registry.redhat.io/ubi9/ubi:latest
-      RUN ls -la /etc/pki/entitlement
+      FROM registry.access.redhat.com/ubi8/ubi:8.7-1112
       RUN rm /etc/rhsm-host
-      RUN yum repolist --disablerepo=*
-      RUN subscription-manager repos --enable rhocp-4.9-for-rhel-8-x86_64-rpms
-      RUN yum -y update
-      RUN yum install -y openshift-clients.x86_64
+      RUN yum --enablerepo=codeready-builder-for-rhel-8-x86_64-rpms \
+          install \
+          sudo \
+          procps-ng \
+          curl \
+          cronie \
+          crontabs \
+          nss_wrapper \
+          uid_wrapper -y && \
+          yum clean all -y
   strategy:
     type: Docker
     dockerStrategy:
+      from:
+        kind: DockerImage
+        name: registry.access.redhat.com/ubi8/ubi:8.7-1112
```

Comment:
I don't suggest using an example that pins to a specific image. This image, [8.7-1112](https://catalog.redhat.com/software/containers/ubi8/5c647760bed8bd28d0e38f9f?architecture=amd64&image=64226b8a5f05d70b2f432ce2&container-tabs=security), is already quite old and missing a number of security fixes. Using `ubi9/ubi:latest` is likely to be a safer, better example.

Reply:
HI @adambkaplan and @control-d . I need some help in better understanding what example we can provide, for customers. What do you think would be a good generic example to present in our docs?